cbauthx/revrpcclient_test.go

```diff
@@ -0,0 +1,38 @@
+package cbauthx
+
+/*
```

Comment:
hmm

Reply:
Ahh, forgot to remove this from the commit.